config.yml

```diff
@@ -4,41 +4,54 @@
 # KEEP_ENVIRONMENTS=OnFail
 # APPS_CHAINLINK_VERSION=1.0.0
 
-network: "ethereum_hardhat" # Selected network for test execution
+network: "ethereum_geth" # Selected network for test execution
 keep_environments: Never # Options: Always, OnFail, Never
 
 apps:
     chainlink:
-        version: 0.10.10
+        image: public.ecr.aws/chainlink/chainlink
+        version: develop.a87e3d12686e0c157aa85cc557f66f0ebca0df25
```

Comment:
I wouldn't lock root config to a develop commit version, but I'm presuming you did this to avoid tests failing due to bugs in 0.10.10? I'd always keep locked to a release version, then just use env vars to run locally with the develop version.